lib/maintainers.nix

```diff
@@ -272,6 +272,7 @@
   ivan-tkatchev = "Ivan Tkatchev <tkatchev@gmail.com>";
   j-keck = "Jürgen Keck <jhyphenkeck@gmail.com>";
   jagajaga = "Arseniy Seroka <ars.seroka@gmail.com>";
+  jthompson = "James Thompson <jamesthompsonoxford@gmail.com>";
```

Comment:
Your handle here should be the same as your github username.

Reply:
Changed @vyp - Thanks for the heads up.